shell/gpu/gpu_surface_software_delegate.h

```diff
@@ -51,7 +51,7 @@ class GPUSurfaceSoftwareDelegate : public GPUSurfaceDelegate {
   virtual bool PresentBackingStore(sk_sp<SkSurface> backing_store) = 0;
 
   // |GPUSurfaceDelegate|
-  ExternalViewEmbedder* GetExternalViewEmbedder() override;
+  virtual ExternalViewEmbedder* GetExternalViewEmbedder() override;
```

Comment:
None of these need to be virtual, the override already implies virtual

I didn't realize thats what you meant before